backend/mlarchive/archive/api.py

```diff
@@ -388,6 +388,11 @@ def post(self, request, **kwargs):
             message['content'] = msg_obj.get_body()
             message['message_id'] = msg_obj.msgid
             message['url'] = msg_obj.get_absolute_url_with_host()
-            message['date'] = msg_obj.date.isoformat()
+            # message date is stored as timezone aware UTC
+            # for simplicity omit the +00:00 offset in the date field
+            # by first converting to a naive datetime
+            dt = msg_obj.date
```

Comment:
One last suggestion, which I admit comes from a deep-seated paranoia: even though we know that the tzinfo attached to the datetime is UTC, go ahead and make the code absolutely explicit about what you mean here. This suggestion will make your code do the right thing for any tzinfo (and even for an incoming naive datetime, as long as it represents a time in the system time zone).
```suggestion
            dt = msg_obj.date.astimezone(datetime.datetime.utc)
```

Reply:
Fixed in commit f822e1c